### ISSUE TITLE
Editing ConfigMap doesn't work from details

### ISSUE THREAD

**Issue by jansav** [kontena/lens#5926]
**Describe the bug**
When trying to edit ConfigMap data in the details and then clicking save, it says that ConfigMap was updated, but it wasn't.

**Comment by jim-docker** [comment]
worked ok for me, using the sample ConfigMap from the dock: '+' -> Create Resource -> Select Template -> ConfigMap

**Comment by jansav** [comment]
> worked ok for me, using the sample ConfigMap from the dock: '+' -> Create Resource -> Select Template -> ConfigMap

Indeed, seems to be working for that sample ConfigMap. Not with "real-one" though. Not sure how that ConfigMap was created. Can't post screenshot / video since it's certificate.

**Comment by anieo** [comment]
lens can't edit/save empty config-maps and secrets for some reason 

you need to edit it from `kubectl edit ` first then you can modify it as you like from lens and add more items in the data map

it maybe related to the fact that when you try to edit an empty secret:
lens show : 
```YAML
data: {}
```
but when I use kubectl, it doesn't show this line

**Comment by yaroslav-metelev** [comment]
@jansav This issue may be related to https://github.com/lensapp/lens/issues/7295
